### DISCUSSION TITLE
Add Support for Forget

### DISCUSSION THREAD

**Discussion by dbpolito** [spatie/once#91] [Ideas]
Would be pretty useful to be able to forget a cache for a class, for my use case i would like to clear the cache for a Eloquent model when refresh is called...

I'm totally open to propose a PR for this, just wanted to discuss about it before i do as there was a similar PR that got rejected:

https://github.com/spatie/once/pull/51

**Comment by freekmurze** [comment]
If the solution is simple, well documented and tests, I'd be open for a PR that adds this feature.
